bcda/models/models.go

```diff
@@ -65,6 +65,7 @@ type Job struct {
 	JobCount          int
 	CompletedJobCount int
 	JobKeys           []JobKey
+	Priority          int
```

Comment:
This is unnecessary, right?  The `Priority` should be on the `que.Job` table already, which you are already setting correctly below.